### ISSUE TITLE
can't open the swagger, when make new angular app

### ISSUE THREAD

**Issue by abdalhmeedshishani** [abpframework/abp#21768]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

I'm using abp version 9.0, but the problem is when ever i create a new app the swagger doesn't work, this error show up ![Screenshot 2024-12-29 141442](https://github.com/user-attachments/assets/061ff1c0-c13c-43c2-b92d-5aa6e28a4413) i can't open the swagger page at all, if someone can help, i just made the app, and install the libs, and after that i run the backend app to test the swagger, and it wasn't working, i believe this is because of the new studio cli.

### Actual behavior

Refused to apply style from 'https://localhost:44389/global-scripts.js?_v=638710662962456189' because its MIME type ('text/javascript') is not a supported stylesheet MIME type, and strict MIME checking is enabled

### Regression?

when i change the cli to studio one from old cli to new one, the swagger is no longer opening

### Version

9.0

### User Interface

Angular

### Database Provider

EF Core (Default)

### Tiered or separate authentication server

None (Default)

### Operation System

Windows (Default)

### Other information

I think problem is from the new cli

**Comment by maliming** [comment]
Please try to use latest 9.0.2 to fix this problem.

**Comment by abdalhmeedshishani** [comment]
Yeah I tried that too, and I tried the older version 8.3.0 too, didn't work none of them, I tried on version 7.4.5 with new cli didn't work, with old cli it's work, and I believe that's the newest versions need the new cli right?

**Comment by maliming** [comment]
How can I reproduce this problem?

**Comment by abdalhmeedshishani** [comment]
Uninstall Volo.Abp.Studio.Cli and reinstall it and try to make new application from CMD by using abp new TodoApp -u angular ,and see what will happen with you.
Or if you want we can go session one on one by using any desk.
Question: should I make the new application by using abp studio app, or it has to work with CMD or power shell too?

**Comment by abdalhmeedshishani** [comment]
@maliming i solved the problem by using this url: https://localhost:port/swagger/index.html, thanks for help
